flutter/cpp/flutter/dart_backend_match.cc

```diff
@@ -8,9 +8,10 @@
 
 extern "C" struct dart_ffi_backend_match_result *dart_ffi_backend_match(
     const char *lib_path, const char *manufacturer, const char *model) {
+  LOG(INFO) << "checking backend '" << lib_path << "'...\n";
   ::mlperf::mobile::BackendFunctions backend(lib_path);
   if (*lib_path != '\0' && !backend.isLoaded()) {
-    std::cerr << "backend '" << lib_path << "' can't be loaded\n";
+    LOG(INFO) << "backend '" << lib_path << "' can't be loaded\n";
```

Comment:
The new line character here (and in other lines where it is appended to `LOG`) creates an empty line.

Reply:
Fixed